packages/core/src/utils/CoreHelperUtil.ts

```diff
@@ -275,5 +285,14 @@ export const CoreHelperUtil = {
     }
 
     return false
+  },
+
+  extractEthereumAddress(address: string) {
```

Comment:
Same. You could use `CoreHelperUtil.getPlainAddress` function from `core` package.